### DISCUSSION TITLE
space-x-2 don't work with flex flex-row-reverse?

### DISCUSSION THREAD

**Discussion by fateh-layachi** [tailwindlabs/tailwindcss#7793] [Help] [answered]
I have a flex div and two or more elements inside like:
`div class="flex flex-row-reverse space-x-2 . . ." 
        <div> div 1</div>
        <div> div 2</div>
/div`

I don't see any space between the two elements, but i see it when I delete the flex-row-reverse class.
Does class space-x-2 not work with class flex-row-reverse?

**Comment by RobinMalfait** [comment, marked as answer]
Hey! Thank you for your question!
Much appreciated! 🙏

Make sure to use `space-x-reverse`: https://tailwindcss.com/docs/space#reversing-children-order
